app/models.py

```diff
@@ -775,6 +788,20 @@ def get_link(self):
             _external=True
         )
 
+    def serialize_for_reports(self):
+        serialized = {
```

Comment:
Is there any reason why this does not follow the format above? 

For consistently use the same style i.e. return { your dict } or return serilized